www/commandQueueExecutor.js

```diff
@@ -52,101 +59,70 @@ function execCmd(success, error, pluginName, methodName, args, execOptions) {
         if (methodName === 'resizeMap') {
           _isResizeMapExecuting = false;
         }
+      
+        if (_stopRequested) {
+          return;
+        }
 
-        // Even if the method was successful,
-        // but the _stopRequested flag is true,
-        // do not execute further code.
-        if (!_stopRequested && success) {
+        if (success) {
           var results = Array.prototype.slice.call(arguments, 0);
-          common.nextTick(function() {
+          ((overlay, results) => {
             success.apply(overlay,results);
-          });
-        }
-
-        // Insert small delays only for `map.getMap()`,
-        // because if you execute three or four `map.getMap()` at the same time,
-        // Android OS itself crashes.
-        // In order to prevent this error, insert small delays.
-        var delay = 0;
-        if (methodName === _isWaitMethod) {
-          if (_isWaitMethod === 'getMap' && Date.now() - _lastGetMapExecuted < 1500) {
-            delay = 1500;
-          }
-          _lastGetMapExecuted = Date.now();
-          _isWaitMethod = null;
+          })(overlay, results);
         }
-        setTimeout(function() {
-          _executingCnt--;
-          common.nextTick(_exec);
-        }, delay);
       },
       function() {
         //-------------------------------
         // error callback
         //-------------------------------
+
         if (methodName === 'resizeMap') {
           _isResizeMapExecuting = false;
         }
-        if (!_stopRequested && error) {
+      
+        if (_stopRequested) {
+          return;
+        }
+
+        if (error) {
           var results = Array.prototype.slice.call(arguments, 0);
+          ((overlay, results) => {
+            error(overlay,results);
+          })(overlay, results);
+
           common.nextTick(function() {
             error.apply(overlay,results);
           });
         }
-
-        if (methodName === _isWaitMethod) {
-          _isWaitMethod = null;
-        }
-        _executingCnt--;
-        common.nextTick(_exec);
       },
       pluginName, methodName, args]
   });
 
-  // In order to execute all methods in safe,
-  // the maps plugin limits the number of execution in a moment to 10.
-  //
-  // Note that Cordova-Android drops has also another internal queue,
-  // and the internal queue drops our statement if the app send too much.
-  //
-  // Also executing too much statements at the same time,
-  // it would cause many errors in native side, such as out-of-memory.
-  //
-  // In order to prevent these troubles, the maps plugin limits the number of execution is 10.
-  if (_isExecuting || _executingCnt >= MAX_EXECUTE_CNT || _isWaitMethod || commandQueue.length === 0) {
-    return;
+  if (!_isExecuting) {
+    setTimeout(function() {
+      common.nextTick(_exec);
+    }, 50);
```

Comment:
See below

Reply:
Ah yea, the execCmd usage of the timeout could be removed. But see previous comment for _exec's usage.